content/en/docs/concepts/storage/ephemeral-volumes.md

```diff
@@ -245,7 +244,6 @@ PVCs indirectly if they can create Pods, even if they do not have
 permission to create PVCs directly. Cluster administrators must be
 aware of this. If this does not fit their security model, they have
 two choices:
-- Explicitly disable the feature through the feature gate.
```

Comment:
This is still possible though not encouraged, right?

Reply:
No, it's locked to enabled when the feature graduates to GA.

Reply:
Got it.